src/constants.ts

```diff
@@ -0,0 +1 @@
+export const API_URL = process.env.NEXT_PUBLIC_API_URL
```

Comment:
[See docs about this](https://nextjs.org/docs/app/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser)